superset/assets/javascripts/explorev2/stores/fields.js

```diff
@@ -31,13 +31,15 @@ export const fields = {
     label: 'Datasource',
     clearable: false,
     default: null,
-    choices: [],
+    mapStateToProps: (state) => ({
```

Comment:
this name seems a bit confusing to me since we use the same function name in components to map state from the store to props. could we call this `getChoices`?